lib-impl/src/main/java/org/ergoplatform/appkit/impl/OutBoxBuilderImpl.java

```diff
@@ -44,6 +45,16 @@ public OutBoxBuilderImpl tokens(ErgoToken... tokens) {
         return this;
     }
 
+    @Override
+    public OutBoxBuilder mintToken(ErgoToken token, String tokenName, String tokenDescription, String tokenNumberOfDecimals) {
+        ErgoValue<Coll<Byte>> tokenNameVal = ErgoValue.of(tokenName.getBytes());
```

Comment:
I think encoding should be explicit UTF-8 or how it is specified in EIP4

Reply:
Good catch! Thanks! Fixed.